src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/PemEncoding.cs

```diff
@@ -270,6 +271,145 @@ private static bool IsWhiteSpaceCharacter(char ch)
             // Match white space characters from Convert.Base64
             return ch == ' ' || ch == '\t' || ch == '\n' || ch == '\r';
         }
+
+        internal static int GetEncodedSize(int labelLength, int dataLength)
```

Comment:
As with the previous commits to this file `GetEncodedSize` is a direct lift of the .NET 5 version of this for M.B.C.